notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>maybe add a note what the second argument of <code>open</code> does and why you use <code>with</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>

Reply:
<p>will do</p>